src/views/Affiliates/CriteriaModal.tsx

```diff
@@ -0,0 +1,330 @@
+import styled from 'styled-components';
+import tw from 'twin.macro';
+
+import { IAffiliateStats } from '@/constants/affiliates';
+import { STRING_KEYS } from '@/constants/localization';
+
+import { useStringGetter } from '@/hooks/useStringGetter';
+import { useWalletConnection } from '@/hooks/useWalletConnection';
+
+import breakpoints from '@/styles/breakpoints';
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Dialog, DialogPlacement } from '@/components/Dialog';
+import { Output, OutputType } from '@/components/Output';
+import { AllTableProps, ColumnDef, Table } from '@/components/Table';
+import { TableCell } from '@/components/Table/TableCell';
+import { Tag } from '@/components/Tag';
+import { WithTooltip } from '@/components/WithTooltip';
+
+import { BorderStatCell, StatCell } from './StatBox';
+
+interface ITierDefinition {
+  tier: number | 'vip';
+  requirements: { referredVol: number; staked: number };
+  affiliateEarnRate: string;
+}
+
+export const CriteriaModal = ({
```

Comment:
Lets implement the CriteriaModal via the DialogManager:
https://github.com/dydxprotocol/v4-web/blob/main/src/layout/DialogManager.tsx

So that it would interact well with the other dialogs and also so we can follow the same pattern for all dialogs.

Let me know if you have questions on this!